module/apmgrpc/server.go

```diff
@@ -36,6 +36,8 @@ import (
 
 	"go.elastic.co/apm"
 	"go.elastic.co/apm/module/apmhttp"
+
+	grpc_middleware "github.com/grpc-ecosystem/go-grpc-middleware"
```

Comment:
go-grpc-middleware is currently only used in tests, and I'd like to keep it that way to minimise dependencies added to instrumented applications.

Can you please remove it and introduce a type similar to WrappedServerStream (but unexported)?

Reply:
Sure, I used it, since it was already imported, but have now updated the PR without the import. I also removed the check for if the stream was already a wrappedServerStream, since it's no longer needed when it's not exported.